src/pipeline.rs

```diff
@@ -232,27 +231,19 @@ impl SpecializedMeshPipeline for OutlinePipeline {
             self.mesh_pipeline.view_layout_multisampled.clone()
         }];
         let mut buffer_attrs = vec![Mesh::ATTRIBUTE_POSITION.at_shader_location(0)];
-        let mut vertex_defs = vec![
-            ShaderDefVal::Int(
-                "MAX_CASCADES_PER_LIGHT".to_string(),
-                MAX_CASCADES_PER_LIGHT as i32,
-            ),
-            ShaderDefVal::Int(
-                "MAX_DIRECTIONAL_LIGHTS".to_string(),
-                MAX_DIRECTIONAL_LIGHTS as i32,
-            ),
-        ];
+        let mut vertex_defs = vec![];
         let mut fragment_defs = vec![];
         bind_layouts.push(
             if mesh_layout.contains(Mesh::ATTRIBUTE_JOINT_INDEX)
                 && mesh_layout.contains(Mesh::ATTRIBUTE_JOINT_WEIGHT)
             {
                 vertex_defs.push(ShaderDefVal::from("SKINNED"));
+                vertex_defs.push("MESH_BINDGROUP_1".into());
```

Comment:
The animated_fox example was failing until I added this... 

```
Shader global ResourceBinding { group: 2, binding: 1 } is not available in the layout pipeline layout
```

I referenced [this](https://github.com/bevyengine/bevy/blob/main/crates/bevy_pbr/src/render/skinning.wgsl#L7) a bit and just added this MESH_BINDGROUP_1 here and that was enough to make everything work for that example. 

This is one change that:
1. Not sure if this should be done here
2. Not sure if this works for all cases

Reply:
I looked back at this and it looks completely fine. the draw command in `draw.rs` uses `SetMeshBindGroup<1>`,

bevy puts mesh bindings in 2 because `StandardMaterial` uses 1. this shader def was created specifically for cases like ours where someone wrote their own mesh pipeline that doesn't use `StandardMaterial`. case in point, [you can see its use in the official mesh instancing example](https://github.com/bevyengine/bevy/blob/main/examples/shader/shader_instancing.rs).

the reason this used to work on the previous version is because this module's old shader sets it to 1 manually. I've since replaced this definition with raw imports of `bevy_pbr::skinning` and `bevy_pbr::morph` which need `MESH_BINDGROUP_1` to function properly.